config/pre-cache/overrides.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
What would the workflow be here. 
Admin applies the override ConfigMap then applies a upgrade cr ?

Reply:
Yes